Library/Homebrew/cask/spec/cask/artifact/binary_spec.rb

```diff
@@ -69,16 +83,25 @@
 
   context "binary is inside an app package" do
     let(:cask) {
-      Hbc.load("with-embedded-binary").tap do |cask|
-        shutup do
-          InstallHelper.install_without_artifacts(cask)
+      Hbc::CaskLoader.load_from_string "with-embedded-binary", <<-'EOS'.undent
```

Comment:
Why use <<-'EOS'.undent here and <<-EOS.undent (without quotes around EOS) above?